game/state/gamestate.cpp

```diff
@@ -1215,7 +1229,141 @@ void GameState::updateEndOfDay()
 	Trace::end("GameState::updateEndOfDay::cities");
 }
 
-void GameState::updateEndOfWeek() { luaGameState.callHook("updateEndOfWeek", 0, 0); }
+void GameState::updateEndOfWeek()
+{
+	updateHumanEconomy();
+
+	luaGameState.callHook("updateEndOfWeek", 0, 0);
+}
+
+// Recalculates AI organization and civilian finances, updating budgets and salaries
+void GameState::updateHumanEconomy()
+{
+	// TODO: remove hardcoded references
+	auto humanCity = cities["CITYMAP_HUMAN"];
+
+	humanCity->populationWorking = 0;
+	// Game resets only Government income, it's not right logically but will keep it to match OG
+	organisations["ORG_GOVERNMENT"]->income = 0;
```

Comment:
I dislike using hardcoded string IDs for this kind of thing but can't think of other way right now. Any suggestions to how to do this better with current code?

Reply:
Maybe add to the GameState the IDs of 'Interesting' organisations? (IE "player = ORG_X-COM", "enemy=ORG_ALIENS", player_funder="ORG_GOVERNMENT")

Then people can change that in mods - but it's a little out of scope here - put it down as a TODO, as there's hundreds of other hardcoded ORG_ALIENs/ORG_WHATEVER around the place :)

Reply:
Yes, I have a TODO with this in the beginning of this method.